airbyte-server/src/main/java/io/airbyte/server/converters/ConfigFileArchiver.java

```diff
@@ -109,6 +109,7 @@ public void importConfigsFromArchive(final Path storageRoot, final boolean dryRu
         readConfigsFromArchive(storageRoot, ConfigSchema.SOURCE_CONNECTION, SourceConnection.class);
         readConfigsFromArchive(storageRoot, ConfigSchema.DESTINATION_CONNECTION, DestinationConnection.class);
         readConfigsFromArchive(storageRoot, ConfigSchema.STANDARD_SYNC, StandardSync.class);
+        readConfigsFromArchive(storageRoot, ConfigSchema.STANDARD_SYNC_OPERATION, StandardSyncOperation.class);
```

Comment:
could we add some tests that remind us to handle these when we add new config schemas?